package.json

```diff
@@ -10,8 +10,30 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@gorhom/bottom-sheet": "^4.1.5",
+    "@react-native-async-storage/async-storage": "^1.15.14",
+    "@react-navigation/bottom-tabs": "^6.0.9",
+    "@react-navigation/native": "^6.0.6",
+    "@react-navigation/stack": "^6.0.11",
+    "axios": "^0.24.0",
+    "i18next": "^21.6.3",
+    "lottie-react-native": "^5.0.1",
     "react": "17.0.2",
-    "react-native": "0.66.4"
+    "react-i18next": "^11.15.1",
+    "react-native": "0.66.4",
+    "react-native-appearance": "^0.3.4",
+    "react-native-elements": "^3.4.2",
+    "react-native-gesture-handler": "^2.1.0",
+    "react-native-linear-gradient": "^2.5.6",
+    "react-native-modal": "^13.0.0",
+    "react-native-really-awesome-button": "^1.6.0",
```

Comment:
Buton için paket kullanmasaydınız keşke. Özellikle belirtmiştim ödevlerde basit UI'lar için paket kullanmayın diye.